controllers/dspipeline_controller.go

```diff
@@ -62,10 +62,8 @@ func (r *DSPAReconciler) Apply(owner mf.Owner, params *DSPAParams, template stri
 		return err
 	}
 
-	if err = tmplManifest.Apply(); err != nil {
-		return err
-	}
-	return nil
+	err = tmplManifest.Apply()
```

Comment:
could you simplify this even further with just `return tmplManifest.Apply()`?  Or is that breaking conventions?